app/controllers/suggested_councillors_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SuggestedCouncillorsController < ApplicationController
+  def new
+    @suggested_councillor = SuggestedCouncillor.new
+    @authority = Authority.find(params[:authority_id])
+  end
+
+  def create
+    @suggested_councillor = SuggestedCouncillor.new(suggested_councillor_params)
+    @authority = Authority.find(params[:authority_id])
+    @suggested_councillor.authority= @authority
+
+      if @suggested_councillor.save
+        flash[:notice] = "Thank you"
+        redirect_to authority_url(@authority.short_name_encoded)
```

Comment:
I'm interested in the choice to redirect to the authority page here. What's your thinking @hisayohorie ?

This is an area where having a single commit for this choice, with a description of why it was made, would make understanding the change easier.